app/client/src/tests/unit/App.test.js

```diff
@@ -0,0 +1,153 @@
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import '../index.css';
+import App from '../../App';
+import {BrowserRouter} from 'react-router-dom';
+import UserEvent from '@testing-library/user-event';
+import { unmountComponentAtNode } from 'react-dom';
+
+/*
+  Setup of DOM container
+*/
+
+let container = null;
+
+function containerInit(){
+  container = document.createElement("div");
+  document.body.appendChild(container);
+
+  render( <React.StrictMode>
+    <BrowserRouter>
+    <App/>
+    </BrowserRouter>
+    </React.StrictMode>, container
+  );
+}
+
+function containerRemove(){
+    unmountComponentAtNode(container);
+    container.remove();
+    container = null;
+}
+
+
+
+describe('Testing log in, sign up, register front-end behaviours', () => {
+  
+  beforeEach(() => {
+    containerInit();
+  });
+
+  afterEach(() => {
+    containerRemove();
+  });
+
+  test('sign up page loads with all four fields and four labels', () => {
```

Comment:
I'd love for these strings within test() to be standardized somehow. We can talk about this at a later time.